components/camel-grpc/src/main/docs/grpc-component.adoc

```diff
@@ -108,30 +107,30 @@ Below is a simple synchronous method invoke with host and port parameters
 [source,java]
 -------------------------------------------------------------------------------
 from("direct:grpc-sync")
-.to("grpc://org.apache.camel.component.grpc.PingPong?method=sendPing&host=localhost&port=1000&synchronous=true");
+.to("grpc://localhost:1101/org.apache.camel.component.grpc.PingPong?method=sendPing&synchronous=true");
```

Comment:
Just to avoid confusion, it might be better to use "remotehost:1101", instead of "localhost:1101" when explaining how to use a grpc producer.